### ISSUE TITLE
Try simpler stat model?

### ISSUE THREAD

**Issue by rcorty** [Ruborcalor/Age-Prediction-Via-Blood-Sample-Methylation-Profiles#1]
Did you try a linear regression with Gaussian error on your processed data?  Didn’t see any mention of such a model in the write up.

**Comment by Ruborcalor** [comment]
Thanks for opening this issue. Just to make sure, do you mean introducing Gaussian noise into the training data?

I did not try this approach, but definitely worth implementing. I'll try and add it to the write up.

**Comment by rcorty** [comment]
Nope, I meant doing a linear regression of age on the methylation factors you selected.  I wonder if it will perform better, worse, or the same as the NN.

**Comment by Ruborcalor** [comment]
Yes I forgot to mention the linear regression in my previous comment, but was more so trying to clarify what you meant when you suggested Gaussian error. Do you mean using Gaussian error as the loss function for the linear regression model?

**Comment by rcorty** [comment]
Yes this is the standard linear regression error distribution.
